lib/tandem/tandemSimulator.js

```diff
@@ -119,6 +120,27 @@ exports.make = function(config){
         events.push(currBasal);
       }
     },
+    newDay: function(event) {
+      ensureTimestamp(event);
+      if((currBasal !== null) && (currBasal.deliveryType !== 'suspend')) {
```

Comment:
Translate this into a proper `currStatus` (e.g., `suspended`) and set the `currStatus` with a fn like the `setCurrBasal` function.